src/dotnet/commands/dotnet-uninstall/tool/UninstallToolCommand.cs

```diff
@@ -49,7 +49,7 @@ public override int Execute()
                 throw new GracefulException(LocalizableStrings.UninstallToolCommandOnlySupportsGlobal);
             }
 
-            var packageId = _options.Arguments.Single();
+            var packageId = _options.Arguments.Single().ToLower();
```

Comment:
remember to remove the ToLower after adapt the package id type

Reply:
Yeah, I'll remove this in favor of constructing the appropriate NuGet representation.